plugins/utils/tmux-navigator.nix

```diff
@@ -0,0 +1,42 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+} @ attrs:
+with lib; let
+  cfg = config.plugins.tmux-navigator;
+  helpers = import ../helpers.nix {inherit lib;};
+in {
+  options.plugins.tmux-navigator = {
+    enable = mkEnableOption "Enable Tmux-Navigator (see https://github.com/christoomey/vim-tmux-navigator for tmux installation instruction)";
+
+    package = helpers.mkPackageOption "tmux-navigator" pkgs.vimPlugins.tmux-navigator;
+
+    tmuxNavigatorSaveOnSwitch = helpers.mkNullOrOption lib.types.int ''
+      null: don't save on switch
+      1: update (write the current buffer, but only if changed)
+      2: wall (write all buffers)
+    '';
+
+    tmuxNavigatorDisableWhenZoomed = helpers.mkNullOrOption lib.types.int ''
```

Comment:
Same as the other option

Reply:
here only null or 1 is allowed, should i put helpers.mkNullOrOption (lib.types.enum [1]) ?

Reply:
It's a bit verbose but yes I think it's usefull